Fix #773 tweak overflow menu

PhotoNomad0 opened the pull request:
Fix #773 tweak overflow menu

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="35" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/ts-android/1414)
<!-- Reviewable:end -->

neutrinog commented:
@PhotoNomad0 I don't see any padding on the menu. There should be some as seen below.

![img](https://cloud.githubusercontent.com/assets/323212/10077814/e5278040-62d9-11e5-9a06-0f20543e49ef.png)

PhotoNomad0 commented:
@neutrinog - sorry Joel, no luck here finding a quick solution.  PopupMenu uses it's own unique styles of which there are very few, and of those few only a small subset of them are recognized before API 21 (compatibility library).

Updated Summary:
Was able to get popup menu to overlap the overflow icon in API 21and greater.
Was able to get white background to popup menu working for API 15 and greater.
Could not find way to add padding to menu.
Changed "My Translations" to "Home"

Found a way to build a custom popupMenu(), but that will take some time.  But don't have quick way to fix.  I can come back to this later it you want to invest the time.
